<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmBenutzerAendern</class>
 <widget class="QDialog" name="frmBenutzerAendern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>80</y>
     <width>239</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblVorname">
      <property name="text">
       <string>Vorname:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edtVorname"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblNachname">
      <property name="text">
       <string>Nachname:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edtNachname"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblBenutzername">
      <property name="text">
       <string>Benutzername:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edtBenutzername"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblPasswort">
      <property name="text">
       <string>Passwort:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edtPasswort"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblOrgEinheit">
      <property name="text">
       <string>OrgEinheit:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="cbOrgEinheit"/>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="cbAdmin">
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="cbDeaktiviert">
      <property name="text">
       <string>Deaktiviert</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btnZurueck">
      <property name="text">
       <string>Zurück</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="btnSpeichern">
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbBeenden">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>438</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMenue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>438</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hauptmenü</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>25</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Benutzer bearbeiten&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
